sdk/face/test-resources.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Do we have live tests set up? I think we're missing a tests.yml in this PR

Reply:
Removed this file because we won't support live test for this preview release. I'd like to support it next preview/GA release. Please let me know if it doesn't meet the release requirements.

Reply:
Sounds good